.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+---
+version: 2.1
+
+executors:
+  ruby_executor:
+    working_directory: /mnt/ramdisk
+    docker:
+      - image: cimg/ruby:3.1.2-browsers
+  rails_executor:
+    working_directory: /mnt/ramdisk
+    docker:
+      - image: cimg/ruby:3.1.2-browsers
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: root
+      - image: cimg/postgres:13.4
+        environment:
+          POSTGRES_USER: root
+          POSTGRES_DB: circle-test_test
+          POSTGRES_HOST_AUTH_METHOD: trust
+        command: postgres -c 'fsync=false' -c 'synchronous_commit=off'
+    resource_class: xlarge
+commands:
+  install_system_dependencies:
+    description: "Install system dependencies"
+    steps:
+      - run:
+          name: install system dependencies
+          command: |
+            # The CircleCI environment is based on Debian buster, which used to be called stable Debian, but is now called oldstable.
+            wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+            sudo apt-get update --allow-releaseinfo-change && \
+            sudo apt-get update && \
+            sudo apt-get install -y pdftk ghostscript
```

Comment:
Your app likely doesn't use `pdftk` even though `vita-min` does, so feel free to remove this line.

Reply:
Removed these packages in 91b1c26b4e770646096f471cb1ab701429f3289a as well